Makefile.PL

```diff
@@ -183,7 +183,7 @@ WriteMakefile(
    # 'TEST_REQUIRES'      => { PDL => '2.004' },
   'PREREQ_PM'          => { PDL => '2.004' },
   'LICENSE'      => 'perl',
-  'VERSION_FROM' => 'hdf5.pd',
+  'VERSION_FROM' => 'HDF5.pm',
```

Comment:
My suggestion here was not supposed to be literal. This shows that you need to just not change this, and delete the explicit version-setting in the metadata. What you're telling it to do here is read from a file that doesn't exist until it's created by running `make`, which won't work from a fresh checkout or CPAN distro, because this change breaks the `perl Makefile.PL` stage, which is why the CI is failing.